architecture-building-systems/CityEnergyAnalyst
# I2146 decentralized cooling

this PR resolves #2146 
to test:
1. run `decentralized_building_main.py` with 
    - a cooling case
    - config.data-helper.region = SG
2. run `decentralized_building_main.py` with 
   - a heating case
   - config.data-helper.region = CH
3. check results in `optimization/decentralized`
